frontend/src/components/pages/events/CountdownButton.tsx

```diff
@@ -70,27 +77,19 @@ export const CountdownButton: React.FC<Props> = ({
     };
   });
 
-  const currentTimeParts = Object.keys(timeLeft).filter((interval) => timeLeft[interval] !== 0);
-
-  const translate = (timeWord: string, time: number) => {
-    if (timeWord === "days") return time > 1 ? "dager" : "dag";
-    if (timeWord === "hours") return time > 1 ? "timer" : "time";
-    if (timeWord === "minutes") return time > 1 ? "minutter" : "minutt";
-    if (timeWord === "seconds") return time > 1 ? "sekunder" : "sekund";
-  };
-
-  /**
-   * `timeparts` is a list containing the elements of time that are not 0,
-   * e.g. 3 days, 14 minutes and 3 seconds yields: ["days", "minutes", "seconds"].
-   * The actual time left is stored in the Record<string, number> called timeLeft.
-   *
-   * Shows remaining time until the event opens on the following formats depending on how much time is left:
-   * - XX days and YY hours
-   * - XX hours and YY minutes
-   * - XX minutes  (minutes left >= 10)
-   * - XX minutes and YY seconds (minutes left < 10)
-   */
   const getCurrentTimeLeft = (timeparts: string[]) => {
```

Comment:
If you'd like to move away from timeparts and such, dayjs has utility for [Time to X](https://day.js.org/docs/en/display/to) which will handle a lot of this automatically. Personally, I find the time parts we've used here to be quite confusing. However, if you would like to change that, it would be better suited for a different pull request to limit the scope of the pull requests. Small pull requests = easy to review and stuff is less likely to break.